From 91613155b2b66a1fd25ccec722fadf1f2de86d15 Mon Sep 17 00:00:00 2001
From: Konrad Leszczynski <konrad.leszczynski@intel.com>
Date: Tue, 15 Dec 2015 12:11:07 +0000
Subject: [PATCH 07/12] BACKPORT: usb: gadget: midi: f_midi_alloc() can be
 static

drivers/usb/gadget/f_midi.c:1072:21: sparse: symbol 'f_midi_alloc'
was not declared. Should it be static?
drivers/usb/gadget/legacy/gmidi.c:118:30: sparse: symbol 'fi_midi'
was not declared. Should it be static?
drivers/usb/gadget/legacy/gmidi.c:119:21: sparse: symbol 'f_midi'
was not declared. Should it be static?

Change-Id: I2e719acd0582029dea2543e6ed91ca04d38da39b
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-3355
Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
Signed-off-by: Felipe Balbi <balbi@ti.com>
Signed-off-by: Konrad Leszczynski <konrad.leszczynski@intel.com>
Reviewed-on: https://android.intel.com:443/448834
---
 drivers/usb/gadget/f_midi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/f_midi.c b/drivers/usb/gadget/f_midi.c
index 7f73236..23badf3 100644
--- a/drivers/usb/gadget/f_midi.c
+++ b/drivers/usb/gadget/f_midi.c
@@ -1098,7 +1098,7 @@ static void f_midi_unbind(struct usb_configuration *c, struct usb_function *f)
 	usb_free_all_descriptors(f);
 }
 
-struct usb_function *f_midi_alloc(struct usb_function_instance *fi)
+static struct usb_function *f_midi_alloc(struct usb_function_instance *fi)
 {
 	struct f_midi *midi;
 	struct f_midi_opts *opts;
-- 
1.9.1

